src/generators/utils.py

```diff
@@ -1,8 +1,93 @@
+import logging
+from typing import List, Union
+
+from steamship import Block, MimeTypes, SteamshipError
+
+
 def safe_format(text: str, params: dict) -> str:
     """Safely formats a user-provided string by replacing {key} with `value` for all (key,value) pairs in `params`."""
+    logging.info(f"Safe Format Text {text} with {params}")
     ret = text
     for (key, value) in params.items():
         if value is not None:
             ret = ret.replace("{" + key + "}", str(value))
-
     return ret
+
+
+def block_to_config_value(block: Block) -> str:
+    if block.mime_type == MimeTypes.TXT:
+        generated_value = block.raw().decode("utf-8")
+        generated_value = (
+            generated_value.strip().lstrip('"').lstrip("'").rstrip('"').rstrip("'")
```

Comment:
Nit: `strip("\"'")` can replace the l and r strips here, I think, and this could probably come down to `strip("\"'\n\t ")` to remove all of the weirdnesses here.

Reply:
Oh nice! Changed.